README.md

```diff
@@ -2,6 +2,152 @@
 
 A collection of Claude Code plugins to automate and assist with various development tasks.
 
+## 🚀 New Hire Quick Start
+
+### Understanding General vs Team-Specific Resources
+
+Welcome to OpenShift engineering! OpenShift consists of **dozens of teams** working across **many repositories**. This guide focuses on **general OpenShift resources** useful to all teams.
+
+#### General Resources (All Teams)
+
+**1. ai-helpers (THIS REPO) - General Automation**
+- 🤖 Claude Code automation plugins for common tasks
+- 🔧 Workflow commands: JIRA, CI analysis, component health
+- 📝 Tools useful across all OpenShift teams
+- 🎯 **Use for:** JIRA automation, CI debugging, bug tracking
+
+**2. openshift/enhancements - General OpenShift Knowledge**
+- 📚 Enhancement proposals (KEPs) for OpenShift features
+- 🏗️ Architecture docs for how OpenShift works
+- 📖 Process docs for the OpenShift project
+- 🔍 **Use for:** Understanding OpenShift design, cross-component architecture
+
```

Comment:
`openshift/enhancements` is definitely a good starter. I'm thinking about what other candidate(s) we could include as "General Resources" here,

- [openshift/release](https://github.com/openshift/release): CI pipeline configurations, reusable scripts
- [openshift/origin](https://github.com/openshift/origin): E2E tests in Golang
- [openshift/openshift-docs](https://github.com/openshift/openshift-docs): User-facing docs 

Ideally while remaining sufficiently general, it should still be informative enough without eating context too much.